source/BoardingPanel.cpp

```diff
@@ -227,6 +229,15 @@ bool BoardingPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command,
 		// When closing the panel, mark the player dead if their ship was captured.
 		if(playerDied)
 			player.Die();
+		// Handle any profit shares that are owed.
+		if(profitShares)
+		{
+			Messages::Add(("You must pay " + Format::Number(profitShares)
+				+ " credits in profit shares to the ")
+				+ (((you->Crew() - 1) > 1) ? "crew members who took part in this boarding action."
+					: "crew member who took part in this boarding action."));
```

Comment:
```suggestion
			Messages::Add(
				"You must pay " +
				Format::Number(profitShares) +
				" credits to your fleet as profit shares."
			);
```

Reply:
Sure, that language suits either model, but I liked having a specific mention of the boarding action for clarity.